packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.0]
+### Added
+- Add optional `getLastBlockVariations` method to `RemoteTransactionSource` type ([#1668](https://github.com/MetaMask/core/pull/1668))
+- Add `updateTransactionGasFees` public method to `TransactionController` ([#1674](https://github.com/MetaMask/core/pull/1674))
+- Add `r`, `s` and `v`  properties to `TransactionMeta` ([#1664](https://github.com/MetaMask/core/pull/1664))
+- Add `sendFlowHistory` property to transaction meta. ([#1665](https://github.com/MetaMask/core/pull/1665))
+- Add `updateTransactionSendFlowHistory` function to the public API of the controller. ([#1665](https://github.com/MetaMask/core/pull/1665))
+- Persist `originalGasEstimate` in transaction metadata ([#1656](https://github.com/MetaMask/core/pull/1656))
+
+### Changed
+- **BREAKING**: Remove `apiKey` property from `RemoteTransactionSourceRequest` type ([#1668](https://github.com/MetaMask/core/pull/1668))
```

Comment:
```suggestion
- **BREAKING**: Remove `apiKey` property from `RemoteTransactionSourceRequest` type ([#1668](https://github.com/MetaMask/core/pull/1668))
  - Not currently used in either client.
```

Reply:
AFAIK we don't reference usage in _clients_(referring to the extension or mobile for example), as a client could be any lib dependant on this package